docs/source/tutorial-tracking-ui.rst

```diff
@@ -0,0 +1,144 @@
+
+If you do not have an instance of the MLflow Tracking Server already running:
+
+1. Navigate to the directory containing the ``mlruns/`` subdirectory you want to visualize. For instance, the Quickstarts and :ref:`tutorial-tracking` expect you to be in the **examples/hyperparam** subdirectory of the cloned MLflow repository.
+2. Run ``mlflow ui``. Use the ``--port`` argument to change from listening on the default port of 5000. Use the same ``--backend-store-uri`` and ``--artifacts-destination`` arguments you used when performing your experiments and using the Tracking API.
+3. Open a browser and navigate to the URL on which you ran the Tracking UI server. For instance, if you used the default port, you would navigate to ``http://localhost:5000``.
+
+The Tracking UI Main page
+-------------------------
+
+When you navigate to the Tracking UI, you will see a page similar to this:
+
+.. image:: _static/images/ui-tutorial/tracking_ui.png
+   :width: 100%
+
+Down the left-hand side of the browser, the UI lists the **Experiments** that are being tracked (1). Individual **Runs** are shown in the main body of the page (2). If you ran either the Quickstarts or the :ref:`tutorial-tracking`, you will see both the parent run, which was the full hyperparameter search, and the child runs, each with unigue parameters.
+
+You can use the **Columns** dropdown (3) to customize the columns displayed in the main results table. You can switch between a **Table view** and a **Chart view** summary of runs (4). The search box (5) allows you to rapidly filter the displayed runs. The **Models** tab (6) displays the registered models that are tracked.
+
+Customize the displayed columns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The **hyperparam** example varied learning rate and momentum while trying to optimize Root Mean Square Error (RMSE) on the validation set. Select the **Columns** dropdown and put checkmarks on the following columns:
+
+* Metrics | val_rmse
+* Parameters | lr
+* Parameters | momentum
+
+Close the dropdown. You may have to horizontally scroll the runs table, but you can now see the chosen parameters and resulting metrics for each run. Usually, the best RMSE is slightly above ``0.49``. Note that the first run in the sweep has an ``lr`` of ``0``, so the model does not train beyond its random initialization; this usually results in an RMSE around ``0.89``.
+
+You can sort on a column by clicking on the column-head. For instance, clicking on the ``val_rmse`` column-head is a quick way to find your best or worse run. Clicking again reverses the sort order.
+
+Compare runs visually with the Chart view
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While you may have a good idea of the best run from the **Table view**, it is often useful to compare runs visually. Select the **Chart view** tab to switch to the chart view.
```

Comment:
or when there are a LOT of runs (running hyperparameter tuning in a staging environment when tuning, I've generated 10's of thousands of runs in a single experiment :) ) definitely not something that I want to read in table format!